### DISCUSSION TITLE
Add jira-cli to winget

### DISCUSSION THREAD

**Discussion by gtbuchanan** [ankitpokhrel/jira-cli#596] [Ideas]
It would be nice to be able to install jira-cli via [winget](https://github.com/microsoft/winget-cli) since it is now bundled with Windows. They now support [portable ZIPs](https://github.com/microsoft/winget-cli/pull/2500). New packages may be submitted [here](https://github.com/microsoft/winget-pkgs).

**Comment by MilanJa** [comment]
https://goreleaser.com/customization/winget/
Seems it's just a matter of adjusting the .goreleaser.yml file?

**Comment by arvinyorro** [comment]
Please release to Winget. I have problems with scoop.

**Comment by ankitpokhrel** [comment]
PR would be nice for this :)
